### DISCUSSION TITLE
how do you use fields with `scope=none`

### DISCUSSION THREAD

**Discussion by macMikey** [serverpod/serverpod#2806] [Q&A] [answered]
if a field is declared with `scope=none`, how do you use it?
i get the "why" part

**Comment by SandPod** [comment, marked as answer]
This is mostly used internally by Serverpod to support implicit relations.

This should add little value for a developer.

**Comment by macMikey** [comment]
it might be good to say exactly that, on the [documentation page](https://docs.serverpod.dev/concepts/models#limiting-visibility-of-a-generated-class).

**Reply by SandPod** [reply]
Good point! Feel free to make an addition to our documentation on this if you have the time: https://github.com/serverpod/serverpod_docs

**Comment by macMikey** [comment]
hope i did that correctly.
https://github.com/serverpod/serverpod_docs/pull/170

**Reply by SandPod** [reply]
Yes, thank you for the contribution! 🙏 

Feel free to keep these coming if you find more things that are strange or hard to understand.

**Comment by macMikey** [comment]
i actually have another one, that i'm in a conversation about with @Isakdl (#2814).
once he has broken through my skull, i'll submit a PR.
